### PR TITLE
Clean up bot config with descriptions for existing rules

### PR DESCRIPTION
**Pull request by jonthysell**
## Description

It's very hard to parse all of our various repo bot rules (like auto triaging, etc). This PR adds a description to each rule and also removed a duplicate rule.

### Type of Change
_Erase all that don't apply._
- New feature (non-breaking change which adds functionality)

### Why
To make the bot rules easier to read, and therefore maintain and extend

### What
See above.

## Screenshots
N/A

## Testing
N/A

## Changelog
Should this change be included in the release notes: no

### CODE REVIEW

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -480,15 +473,14 @@ configuration:
       then:
       - addLabel:
           label: 'Partner: Microsoft'
-      description: 
-    - if:
+    - description: "Close issues resolved as won't fix"
```
Comment (marlenecota) [inline .github/policies/resourceManagement.yml:476]: Is this in quotes cuz of the apostrophe?

File: .github/policies/resourceManagement.yml [hunk 2]
```diff
@@ -364,8 +357,8 @@ configuration:
           label: 'Needs: Author Feedback'
       - removeLabel:
           label: 'Needs: Triage :mag:'
-      description: 
-    - if:
+    - description: Mark new issues as coming from Facebook when the author is from Facebook
```
Comment (marlenecota) [inline .github/policies/resourceManagement.yml:360]: Meta?

Comment (jonthysell) [reply .github/policies/resourceManagement.yml:360]: If and when we rename the label the description can be renamed too.

**Review by marlenecota — APPROVED** [review]
